### DISCUSSION TITLE
Live Transcript Streamed To Website

### DISCUSSION THREAD

**Discussion by nccscott** [deepgram/community#373] [General help] [answered]
I am very new to using Deepgram and other APIs.  We are looking to make a live transcription device for live events.  We would like to have a simple device like a raspberry pi 4 connected to our soundboard.  Then have the device transcribe the speaker.  Then have that live transcription displayed on a website that people can access via their own device to see the transcription being sent by the raspberry pi.

Any help would be much appreciated!  Thank you all

**Comment by SandraRodgers** [comment, marked as answer]
Hi @nccscott,

Have you seen [this resource](https://deepgram.com/learn/live-transcription-badge-video)? I highly recommend watching the video, which walks you through using a raspberry pi to build a live transcription device similar to what you're trying to build.

We also have a lot of resources that help with displaying captions on a website. Here are a few:
- https://github.com/deepgram-devs/node-live-example
- https://deepgram.com/learn/live-transcription-mic-browser
- https://deepgram.com/learn/classroom-captioner

**Comment by nccscott** [comment]
I was actually able to follow the instructions and get this working.  Is there a way to get it to display the captions to someone who is not sending the audio data to be transcribed?  For example, device A (raspberry pi) captures the audio and transcribes it by our soundboard.  Then someone on device B (smart phone) goes to a website to see the transcription being produced by device A?

Thank you for all the helpful links, I will look into them as well.  thank you



On Oct 12, 2023, at 1:07 PM, Sandra Rodgers ***@***.***> wrote:



Hi @nccscott<https://github.com/nccscott>,

Have you seen this resource<https://deepgram.com/learn/live-transcription-badge-video>? I highly recommend watching the video, which walks you through using a raspberry pi to build a live transcription device similar to what you're trying to build.

We also have a lot of resources that help with displaying captions on a website. Here are a few:

  *   https://github.com/deepgram-devs/node-live-example
  *   https://deepgram.com/learn/live-transcription-mic-browser
  *   https://deepgram.com/learn/classroom-captioner
